viz_config.yml

```diff
@@ -3,24 +3,31 @@ storm_name: "Hurricane Harvey"
 dates:
   start: '2017-08-25'
   end: '2017-09-02'
+
+# view, dimensions
 bbox: [-100, 22, -80, 34]
 projection: "+proj=laea +lat_0=45 +lon_0=-100 +x_0=0 +y_0=0 +a=6370997 +b=6370997 +units=m +no_defs"
 height: 512
 width: 1024
+
+# styling
 precip_cols: ["#FAFBF3", "#F0F8E3", "#D4E9CA", "#BBE0CE", "#B7DAD0", "#B0CCD7", "#A9B8D7", "#A297C2", "#8F6F9E", "#684A77", "#41234D"]
 precip_bin: 4 # inches per bin
 gage_flood_col: "#175676"
 gage_norm_col: "#4BA3C3"
 hurricane_col: "orange"
 hurricane_cols: ["#71c633", "#f4f141", "#f47c41", "#f44141", "#d63995"]
 hurricane_col_names: ["Tropical Storm", "1", "2", "3", "4"] # If these change, legend layout may break
-n_intro_frames: 20 # number of frames in the intro animation linking gages to sparklines
+
+# introductory animation
+intro:
+  t_last_start: 5 # last frame in the intro animation when a site should leave its lat/lon (they're staggered)
+  n_frames: 20 # number of frames in the intro animation linking gages to sparklines
+  elbow_radius: 50000 # radius of the curvature at the elbow of the dot travel path, in 10000ths of degrees
+  point_background: "#175676"
```

Comment:
this is the same as `gage_flood_col` shouldn't the default background be the same color as `gage_norm_col`?

Reply:
Good question worthy of discussion. I played with having it be `gage_norm_col`, but then in the first few frames of the animation I had a hard time visually pickout out which dots were the true gage locations and which dots were the moving gage symbols. The darker blue is a bit nicer for that and doesn't introduce any big color distractions as a truly new color would (I tried some purples, too)...but it's entirely possible we could do still better by picking some other color. I extracted this parameter as something we could set directly in the yml because I do hope we'll play with some options.